CairoMakie/test/svg_tests.jl

```diff
@@ -70,3 +70,10 @@ end
     rm("test1.svg")
     rm("test2.svg")
 end
+
+@testset "issue 4970" begin
+    @test_nowarn begin
+        sprint(io -> show(io, "image/svg+xml", Scene()))
+        GC.gc()
+    end
+end
```

Comment:
Let's test this for all mimes!

```suggestion
@testset "issue 4970" begin
    for mime in CairoMakie.SUPPORTED_MIMES
        @testset "$mime" begin 
            @test_nowarn begin
                sprint(io -> show(io, mime, Scene()))
                GC.gc(true)
           end
        end
    end
end
```

Reply:
moved the test to another file with the loop